### ISSUE TITLE
Android instrumented tests broken from 1.0.4 onwards (1.0.3 works great)

### ISSUE THREAD

**Issue by Buggaboo** [xtext/xtext-gradle-plugin#69]
```
./gradlew cAT // plugin version 1.0.9
./gradlew cDAT // plugin version 1.0.9
```

fails with:

```
:Xtendroid:preBuild
...
:XtendroidTest:generateDebugAndroidTestXtext FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':XtendroidTest:generateDebugAndroidTestXtext'.
> java.io.FileNotFoundException: /Users/user/Documents/git-repositories/Xtendroid/XtendroidTest/build/intermediates/classes/debug (Is a directory)

* Try:
Run with --info or --debug option to get more log output.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':XtendroidTest:generateDebugAndroidTestXtext'.
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:69)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:46)
	at org.gradle.api.internal.tasks.execution.PostExecutionAnalysisTaskExecuter.execute(PostExecutionAnalysisTaskExecuter.java:35)
	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:66)
	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:58)
	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:52)
	at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:52)
	at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:53)
	at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
	at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:203)
	at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:185)
	at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker.processTask(AbstractTaskPlanExecutor.java:66)
	at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker.run(AbstractTaskPlanExecutor.java:50)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:54)
	at org.gradle.internal.concurrent.StoppableExecutorImpl$1.run(StoppableExecutorImpl.java:40)
Caused by: org.gradle.api.UncheckedIOException: java.io.FileNotFoundException: /Users/user/Documents/git-repositories/Xtendroid/XtendroidTest/build/intermediates/classes/debug (Is a directory)
	at org.gradle.internal.UncheckedException.throwAsUncheckedException(UncheckedException.java:43)
	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:78)
	at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory$IncrementalTaskAction.doExecute(AnnotationProcessingTaskFactory.java:245)
	at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory$StandardTaskAction.execute(AnnotationProcessingTaskFactory.java:221)
	at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory$IncrementalTaskAction.execute(AnnotationProcessingTaskFactory.java:232)
	at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory$StandardTaskAction.execute(AnnotationProcessingTaskFactory.java:210)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:80)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:61)
	... 14 more
Caused by: java.io.FileNotFoundException: /Users/user/Documents/git-repositories/Xtendroid/XtendroidTest/build/intermediates/classes/debug (Is a directory)
	at com.google.common.io.Files$FileByteSource.openStream(Files.java:126)
	at com.google.common.io.Files$FileByteSource.openStream(Files.java:116)
	at com.google.common.io.ByteSource.copyTo(ByteSource.java:247)
	at com.google.common.io.ByteSource.hash(ByteSource.java:325)
	at com.google.common.io.Files.hash(Files.java:604)
	at org.xtext.gradle.builder.XtextGradleBuilder$3.apply(XtextGradleBuilder.java:231)
	at org.xtext.gradle.builder.XtextGradleBuilder$3.apply(XtextGradleBuilder.java:226)
	at org.eclipse.xtext.xbase.lib.IteratorExtensions.forEach(IteratorExtensions.java:363)
	at org.eclipse.xtext.xbase.lib.IterableExtensions.forEach(IterableExtensions.java:333)
	at org.xtext.gradle.builder.XtextGradleBuilder.indexChangedClasspathEntries(XtextGradleBuilder.java:280)
	at org.xtext.gradle.builder.XtextGradleBuilder.build(XtextGradleBuilder.java:109)
	at org.xtext.gradle.tasks.XtextGenerate.generate(XtextGenerate.java:115)
	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:75)
	... 20 more


BUILD FAILED
```

The tasks `connectedAndroidTest` (cAT) and  `connectedDebugAndroidTest` (cDAT), all work perfectly on 1.0.3. Every version after has the same result as I posted up here.

I tested with 1.0.9, because this is the latest version that still works for the Xtendroid project (without connected tests).

**Comment by oehme** [comment]
This is fixed in 1.0.12.

**Comment by Buggaboo** [comment]
Cool, I'm looking forward to 1.0.13.
